doc/api/net.md

```diff
@@ -7,35 +7,53 @@ functions for creating both servers and clients (called streams). You can includ
 this module with `require('net');`.
 
 ## Class: net.Server
+<!-- YAML
+added: v0.7.5
+-->
 
 This class is used to create a TCP or local server.
 
 `net.Server` is an [`EventEmitter`][] with the following events:
 
 ### Event: 'close'
+<!-- YAML
+added: v0.9.12
```

Comment:
I think this one must be older too, seems to have been introduced in 8bf5b8d352286